webClient/src/app/editor/project-tree/project-tree.component.ts

```diff
@@ -166,6 +171,15 @@ export class ProjectTreeComponent implements OnInit {
   }
   
   ngOnInit() {
+    this.subscription.add(this.appKeyboard.keyupEvent.subscribe((event) => {
+      console.log('appKeyboard keyup', event);
```

Comment:
this needs to be log.debug, probably, but at least not console.log.

Reply:
modified to log.debug